.github/workflows/mega-linter.yml

```diff
@@ -125,7 +125,9 @@ jobs:
             (
               github.event_name == 'push' ||
               github.event.pull_request.head.repo.full_name == github.repository
-            )
+            ) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+
```

Comment:
Included in equivalent https://github.com/oxsecurity/megalinter/pull/3032/files#diff-206ce5348d308a71dab8cdc132d6dc2babebc80b94a707a45a744909ed9367d4R286-R287